src/regressor.jl

```diff
@@ -57,11 +57,22 @@ MultivariateNeuralNetworkRegressor(; builder::B   = Linear()
                                        , alpha
                                        , optimiser_changes_trigger_retraining)
 
+const Regressor =
+    Union{NeuralNetworkRegressor, MultitargetNeuralNetworkRegressor}
+
+function nrows(X)
```

Comment:
Maybe it's better to shift the definition of `nrows` to src/core.jl, given that it can be reused for collate  in src/classifier.jl?